### PR TITLE
Utilized `nameof()` for `ArgumentNullException`s.

### PR DESCRIPTION
**Pull request by henkmollema**
Utilized the use of `nameof()` operator for `ArgumentNullException`s.

### CODE REVIEW

File: src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryOptions.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ public string CookieName
             {
                 if (value == null)
                 {
-                    throw new ArgumentNullException("value",
+                    throw new ArgumentNullException(nameof(value),
                                                     Resources.FormatPropertyOfTypeCannotBeNull(
                                                                 "CookieName", typeof(AntiForgeryOptions)));
```
Comment (kichalla) [inline src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryOptions.cs]: nameof(CookieName)

File: src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryOptions.cs [hunk 2]
```diff
@@ -61,7 +61,7 @@ public string FormFieldName
             {
                 if (value == null)
                 {
-                    throw new ArgumentNullException("value",
+                    throw new ArgumentNullException(nameof(value),
                                                     Resources.FormatPropertyOfTypeCannotBeNull(
                                                                 "FormFieldName", typeof(AntiForgeryOptions)));
```
Comment (kichalla) [inline src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryOptions.cs]: nameof(FormFieldName)

File: src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs [hunk 3]
```diff
@@ -50,7 +50,7 @@ public AntiForgeryOptions AntiForgeryOptions
             {
                 if (value == null)
                 {
-                    throw new ArgumentNullException("value",
+                    throw new ArgumentNullException(nameof(value),
                                                     Resources.FormatPropertyOfTypeCannotBeNull("AntiForgeryOptions",
```
Comment (kichalla) [inline src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs]: nameof(AntiForgeryOptions)

**Comment by henkmollema** [comment]
@kichalla addressed feedback.

**Comment by kichalla** [comment]
:shipit:

**Comment by Eilon** [comment]
@kichalla please merge.

**Comment by kichalla** [comment]
955b45f995df0fa7a8a43641d979ae4184c733ac

**Pull request closed without merging**